bootstrap/bootstrap.go

```diff
@@ -1308,7 +1308,7 @@ func (b *Bootstrap) defaultCheckoutPhase(ctx context.Context) error {
 	var mirrorDir string
 
 	// If we can, get a mirror of the git repository to use for reference later
-	if experiments.IsEnabled(`git-mirrors`) && b.Config.GitMirrorsPath != "" && b.Config.Repository != "" {
```

Comment:
aha, i was foolish in searching only for `"git-mirrors"`. nice catch!

Reply:
Sorry, bad habit of mine is using `` for const-ish strings.